src/gluonnlp/sequence_sampler.py

```diff
@@ -286,22 +285,26 @@ def gumbel_with_maximum(self, phi, T, dim=-1):
         g = self.shift_gumbel_maximum(g_phi, T, dim, Z=Z)
         return g
 
-    def shift_gumbel_maximum(self, g_phi, T, dim=-1, Z=None):
+    def shift_gumbel_maximum(self, g_phi, T, axis=-1, Z=None):
         """
+
         Parameters
         ----------
-        F
         g_phi : mx.np.ndarray
             Shape (batch_size, beam_size, L).
         T : mx.np.ndarray
             The previous scores. Shape (batch_size, beam_size)
+        axis
+            The axis
+        Z
+            The Z value
```

Comment:
we can probably make these descriptions more helpful. the docstring above may also help.